README.md

```diff
@@ -136,21 +156,45 @@ or stopped as well as monitored for a dynamic lifecycle
 management.
 
 
-##### coap
+#### comms
+
+Comms consists on a few communication modules.
+It provides ways to deal with network, CoAP protocol and
+OIC protocol (server and client sides).
+
+
+##### Network
+
+Network library provides a way to handle network link interfaces,
+making it possible to observe events, to inquire available links
+and to set their states.
+
+##### COAP
 
 Implementation of The Constrained Application Protocol (CoAP -
 RFC 7252). This network protocol is pretty simple and uses a
 HTTP-like message that is space-efficient over UDP.
 
+##### OIC
+
+Implementation of protocol defined by Open Interconnect Consortium
+(OIC - http://openinterconnect.org/)
```

Comment:
It creates links automagically:
Please check it on https://github.com/bdilly/soletta/